packages/core/src/oidc/init.ts

```diff
@@ -198,6 +209,63 @@ export default function initOidc(envSet: EnvSet, queries: Queries, libraries: Li
       },
     },
     extraParams: [OIDCExtraParametersKey.InteractionMode],
+    extraTokenClaims: async (ctx, token) => {
+      try {
+        const { isDevFeaturesEnabled, isCloud } = EnvSet.values;
+        // No cloud connection for OSS version, skip.
+        if (!isDevFeaturesEnabled || !isCloud) {
+          return;
+        }
```

Comment:
nit:  
- could move the devFeature and cloud check outside of the try..catch. 
- extract the getCustomJwtClaims as a method, and put the try-catch inside that method. 
easy to manage. In case we need to add other logto token claims logic in the future.

Reply:
1. Moved them outside try catch.
2. so we can do this later. For now, if we wrap this as a util method, we need to pass tons of input parameters to it, including query and library methods, token, cloud connection methods. I do not think this can make the implementation simpler.